examples/rbd/storageclass.yaml

```diff
@@ -73,8 +73,16 @@ parameters:
    # By default host-path /var/log/ceph of node is bind-mounted into
    # csi-rbdplugin pod at /var/log/ceph mount path. See docs/rbd-nbd.md
    # for available configuration options.
+   # This is to configure target bindmount path used inside container.
```

Comment:
>This is to configure target bindmount path used inside container

This does not reflect that this is for logging?

add a note this is only for rbd-nbd?